create-frog/create.ts

```diff
@@ -10,6 +10,38 @@ const __dirname = dirname(fileURLToPath(import.meta.url))
 
 export type CreateParameters = { name: string; template: string }
 
+function templateNameToDefaultPort(name: string): number {
+  if (name === 'next')
+    // `pnpm dev` runs `next dev`
+    return 3000
+  if (
+    name === 'bun' ||
+    name === 'node' ||
+    name === 'cloudflare-worker' ||
+    name === 'default' ||
+    name === 'vercel'
+  )
+    // `pnpm dev` runs `frog dev`
+    return 5173
+  throw new Error('Unknown template name')
+}
+
+function templateNameToDefaultFrogPath(name: string): string {
+  if (name === 'next')
+    // frog is served in `/api` route in NextJS instance
+    return '/api'
+  if (
+    name === 'bun' ||
+    name === 'node' ||
+    name === 'cloudflare-worker' ||
+    name === 'default' ||
+    name === 'vercel'
+  )
+    // frog instance is run via `frog dev`
+    return '/dev'
```

Comment:
```suggestion
  switch(name) {
    case 'next'
      return '/api'
    default:
      return '/dev'
  }
```

Reply:
> 

I still believe it is important to throw on unknown template name as such might have a different port and endpoint.
But will refactor to switch clause!